packages/preview/clean-hda/0.1.0/README.md

```diff
@@ -0,0 +1,293 @@
+
+```typst
+#import "./hda_template/template/main.typ": *
+```
+
+Contributions are welcome.
+
+# Clean HDA
+
+To see the direct changes compared to to [forked DHBW
+project](https://github.com/roland-KA/clean-dhbw-typst-template.git), consider
+taking a look at [CHANGELOG.MD](./CHANGELOG.md).
+
+The following documentation is the original from [roland-KA/clean-dhbw-typst-template](https://github.com/roland-KA/clean-dhbw-typst-template)
+
+# Clean DHBW
+
+A (modernized) [Typst](https://typst.app/) template for DHBW documents like Bachelor theses, "Studienarbeiten", project documentation etc. It is the official Typst template for Computer Science at DHBW Karlsruhe.
+
+You can see an example of how the template looks in this [PDF file](https://github.com/roland-KA/clean-dhbw-typst-template/blob/main/template/main.pdf).
+
+## Introduction and Motivation
+
+In my experience as an end-user (i.e. reader) of documents like Bachelor theses and similar works which are currently produced at DHBW, there is room for improvement with respect to their usability.  There exists a recommendation at DHBW on how to structure and design such documents. I have the impression that some of the usability issues I identified are rooted directly within these recommendations, but others stem from the fact that some recommendations are just not thoroughly followed.
+
+In order to give an example on how an improved document structure and layout could look like, I have created the present "*Clean DHBW Typst Template*". In the meantime it is the official Typst template for Computer Science at DHBW Karlsruhe. But of course anyone interested is welcome to use it too.
+
+Of course such a concept is always a bit biased in some way. Therefore I explain the whys and hows below and I'm looking forward to the discussions it will provoke 😃. The name has been chosen in the sense "clean" is used in Software Engineering in terms like *clean architecture* or *clean code*, where it describes concepts and structures which are easy to understand, to use und to maintain. Furthermore they have a clear separation of concerns and responsibilities – for the case at hand that means: the template defines the typography, whereas the author is responsible for the contents.
+
+There exists already a Typst template for these sorts of documents: It's the ["supercharged-dhbw"-template](https://github.com/DannySeidel/typst-dhbw-template) by Danny Seidel. It is a great piece of work with a lot of functionality covering a broad variety of use cases. But with respect to structure and layout, it implements exactly the above criticized state (which is without doubt what many people want or maybe have to use). I discussed with Danny on how to realize further development. We agreed to keep `supercharged-dhbw` more or less as-is in order to reflect current state and needs and in consequence to build this new template as a fork of his work. This gave me also  more freedom to go new ways.
```

Comment:
This seems to be completely unchanged from the original repository, which can be kind of misleading.
I think in general, you should just link to the original repository (maybe to the specific git revision from where you forked) and only include information that directly relates to this package in this readme. You can of course include a comparison section, but the support section also still links to the original repository.